metricbeat/module/mongodb/status/conversion.go

```diff
@@ -0,0 +1,97 @@
+package status
```

Comment:
+1 on moving it to schema as a top level package as this is so critical to building metricsets. Also then we can use `s.*` instead of `h.*` which somehow feels more logical. Or would you use the full package name every time?